contracts/tests/conftest.py

```diff
@@ -183,36 +183,74 @@ def block_header_by_malicious_non_validator(malicious_non_validator_key):
 
 
 @pytest.fixture(scope="session")
-def real_price_validator_auction_contract(deploy_contract, whitelist):
-    contract = deploy_contract("ValidatorAuction")
+def validator_auction_contract(deploy_contract, whitelist, web3, default_account):
+    deposit_locker = deploy_contract("DepositLocker")
+    contract = deploy_contract(
+        "TestValidatorAuctionFixedPrice", constructor_args=(deposit_locker.address,)
+    )
+    deposit_locker.functions.init(
+        web3.eth.blockNumber + 50,
+        "0x0000000000000000000000000000000000000000",
+        contract.address,
+    ).transact({"from": default_account})
+
     add_whitelist_to_validator_auction_contract(contract, whitelist)
 
     return contract
 
 
 @pytest.fixture(scope="session")
-def no_whitelist_validator_auction_contract(deploy_contract):
-    contract = deploy_contract("TestValidatorAuctionFixedPrice")
+def real_price_validator_auction_contract(
+    deploy_contract, whitelist, default_account, web3
+):
+    deposit_locker = deploy_contract("DepositLocker")
+
+    contract = deploy_contract(
+        "ValidatorAuction", constructor_args=(deposit_locker.address,)
+    )
+    deposit_locker.functions.init(
+        web3.eth.blockNumber + 50,
+        "0x0000000000000000000000000000000000000000",
+        contract.address,
+    ).transact({"from": default_account})
```

Comment:
The default_account is used per default, no need to specify it

Reply:
760af1bab84ca6aa80b60bc59c4761e6e594f822